README.md

```diff
@@ -182,6 +182,31 @@ CoreDNS is a fast and flexible DNS server. It has a focus of service discovery i
 -	Recommended Skills: Golang, Microsoft Azure
 -	Mentor(s): Yong Tang (@yongtang)
 
+### Add annotations to Storyscript
+
+> [Storyscript](https://github.com/storyscript), by [Asyncy](https://asyncy.com) an open-source microservice choreography platform, is a language for stringing microservices together on Kubernetes. Asyncy is a CNCF/LF member.
```

Comment:
@caniszczyk @idvoretskyi Can CNCF/LF members also submit projects for GSoC or is it only the projects listed here https://www.cncf.io/projects?

Reply:
it's only for CNCF projects